### ISSUE TITLE
possible errors in lstm.ipynb

### ISSUE THREAD

**Issue by hua-wu-que** [huseinzol05/Stock-Prediction-Models#76]
there is a line in lstm.ipynb which is:
"self.logits = tf.layers.dense(self.outputs[-1], output_size)"

What is the meaning of select self.outputs[-1]?
It seems you are doing prediction for each time step, 
however the default behavior of dynamic_rnn is time_major=False, 
so self.outputs is of size [batch_size, time_step, state_size]

Can you clarify this?

btw, although in your code it works on. because batch_size = 1.
Thank you.

**Comment by huseinzol05** [comment]
can you paste link to notebook?
